stratus/settings.py

```diff
@@ -130,3 +137,8 @@
 
 LOGIN_REDIRECT_URL = '/home/'
 LOGIN_URL = '/login/'
+
+try:
+    from local_settings import *
+except ImportError:
+    pass
```

Comment:
```suggestion
```
People do this and I have mixed opinions. Django has a `--settings` for a reason, so let's use that.